treeherder/client/thclient/client.py

```diff
@@ -419,22 +426,15 @@ def init_data(self):
     def add_push_timestamp(self, push_timestamp):
         self.data['push_timestamp'] = push_timestamp
 
-    def add_revision_hash(self, revision_hash):
-        self.data['revision_hash'] = revision_hash
-
     def add_author(self, author):
         self.data['author'] = author
 
     def add_revisions(self, revisions):
-        if revisions:
-            self.data['revisions'] = revisions
+        for revision in revisions:
```

Comment:
this has always been broken.  should have been adding the ``.data`` property...  :)